mdct-setup.sh

```diff
@@ -503,9 +503,9 @@ for url in "${repo_urls[@]}"; do
 done
 
 # Install serverless
-if ! which sls > /dev/null ; then
-  echo "npm installing serverless v3.38.0"
-  npm install -g serverless@3.38.0
+if ! which serverless > /dev/null || ! serverless --version 2>&1 | grep -q " 4."; then
```

Comment:
```suggestion
if ! which serverless > /dev/null || ! serverless --version 2>&1 | grep -q "Framework 4."; then
```
Because of this comment: https://github.com/Enterprise-CMCS/macpro-mdct-seds/pull/15041#discussion_r1936036937